firebase-perf/src/main/java/com/google/firebase/perf/FirebasePerfRegistrar.java

```diff
@@ -56,7 +56,6 @@
             // components that are not required during initialization so as not to force initialize
             // them at app startup (refer
             // https://github.com/google/guice/wiki/InjectingProviders#providers-for-lazy-loading).
-            .eagerInDefaultApp()
```

Comment:
Can you update the comment above to reflect that the SDK is no longer eager?

Reply:
Removed.